pkg/client/unversioned/remotecommand/v3.go

```diff
@@ -72,6 +72,9 @@ func (p *streamProtocolV3) handleResizes() {
 
 		encoder := json.NewEncoder(p.resizeStream)
 		for {
+			if p == nil || p.TerminalSizeQueue == nil {
```

Comment:
Good point. I was not sure if we lose this (ends up with nil) in the for loop. but definitely worth starting there